src/app/globals.css

```diff
@@ -2,16 +2,19 @@
 @tailwind components;
 @tailwind utilities;
 
-
 @layer base {
   @font-face {
     font-family: 'Rex-Bold';
     src: url('../styles/fonts/Rex-Bold.ttf');
-    }
+  }
+  @font-face {
```

Comment:
Need to preserve same code style as line 7, so it is `Rex-Inline`. It is all about being consistent🙂